src/data/repack.cc

```diff
@@ -15,20 +15,16 @@
  *
  */
 #include "data/repack.h"
+
 #include <stdlib.h>
 
+#include "util/math.h"
```

Comment:
Do we have any convention about include sorting? In previous commit you moved includes in a way that doesn't match what you do here.